qml.qrc

```diff
@@ -308,6 +308,7 @@
         <file>QML/Assets/items/Inv_pants_02.png</file>
         <file>QML/Assets/items/Inv_pants_06.png</file>
         <file>QML/Assets/items/Inv_pants_12.png</file>
+		<file>QML/Assets/items/Inv_pants_leather_05.png</file>
```

Comment:
I've fixed this in a7390aa5e9aa79ba74d31b75fcf89949c0fc7b29 so you can remove this.